port/win/port_win.h

```diff
@@ -81,8 +81,8 @@ namespace rocksdb {
 
 namespace port {
 
-// VS 15
-#if (defined _MSC_VER) && (_MSC_VER >= 1900)
+// VS 15 / MinGW
+#if !defined(_MSC_VER) || (_MSC_VER >= 1900)
```

Comment:
According to [the docs](https://msdn.microsoft.com/en-us/library/58dt9f24(v=vs.100).aspx) it was only added in MSVC2015.